improver_tests/ensemble_copula_coupling/test_utilities.py

```diff
@@ -444,10 +388,8 @@ def test_percentile_is_dimension_coordinate_multiple_timesteps(self):
         """
```

Comment:
This comment could be made clearer by pointing out that the input here has multiple timesteps and percentiles. The output has a single percentile, which is not promoted to a single-valued leading dimension, so the result has one fewer dimensions than the input. At a glance it looks like a dimension has gone walkies.

Reply:
Ugh you are correct.  I'm also really not happy with these tests, on reflection, since almost all of them provide an input data array that does not actually require reshaping!  There is only one test which in fact provides a flattened input array.  I'm going to modify the inputs to these tests and remove the duplicate which will then result.

Reply:
Ah actually even having done this, this test is mis-describing itself.  It sets up this input cube with 3 percentile points which it never uses!  I'll simplify the setup to create what's actually being used and rename the test, because it is just a "test_multiple_timesteps" - single valued coordinates are scalar.